src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -26,17 +26,16 @@ namespace CoordinateMaps {
  *
  * \details The mapping that goes from a reference cube to a three-dimensional
  *  wedge centered on a coordinate axis covering a volume between an inner
- *  surface and outer surface. One of the surfaces must be spherical, but the
- *  curvature of the other surface can be anything between flat (a sphericity of
- *  0) and spherical (a sphericity of 1).
+ *  surface and outer surface. These surfaces can be independently chosen to be
+ *  flat (a sphericity of 0) or spherical (a sphericity of 1).
```

Comment:
This does not appear to be true, and there are similar inconsistent changes throughout the docs.  I am generally very confused about what is going on with the bounds and sphericities and such in this PR.

Reply:
Good catch. I meant to have `sphericity_outer` be independently specifiable in a second commit and only introduce the variable here with a hard-coded value.